docs/UserGuide.md

```diff
@@ -384,56 +458,146 @@ Format: `c-clear`
+to work again in the future. You, as a store manager, want to remove him from your employee directory without
+permanently deleting his information, so that you can retrieve/move back his information to your
+employee directory when he comes back to work again. In this case, you can follow these steps.
+
+Steps to archive the first employee, Alex Yeoh, in the sample data:
+1. Type `c-archive 1` into the _Command Box_.
+2. Press "Enter" to execute.
+
+
+Outcome:
+1. The _Result Display_ will show a success message with the employee's name that you have archived.
+2. Alex Yeoh's information will be removed from _Employee Directory_ pane.
+
+{Screenshot}
 
 #### 3.11 Archiving all employees : `c-archive-all`
 
-Archives all employees' contact details from the tCheck.
+Archives all employees in the _Employee Directory_ pane.
 
 Format: `c-archive-all`
 
+<div markdown="span" class="alert alert-primary">
+
+**:information_source: Notes about Employee Directory:**<br>
+
+* If _Employee Directory_ currently shows a list of archived employees (archived employees are displayed with a red
+ color label, named as `[Archived]`, in front of their names), command `c-active-list` needs to be used first to see a
+  list of all your employees. Then, `c-archive-all` can be used to archive all employees.<br>
+
+</div>
+
+Example:
+* Let's say after you familiarize yourself with tCheck, you want to remove all sample data in employee directory. But
+ you don't want to permanently delete those data, so that you can use them as a reference in the future. In this case
+ , you can follow these steps to archive all employees.
+
+Steps to archive all employees in the sample data:
+1. Type `c-archive-all` into the _Command Box_.
+2. Press "Enter" to execute.
+
+
+Outcome:
+1. The _Result Display_ will show a success message.
+2. All employees will be removed from _Employee Directory_ pane.
+
+{Screenshot}
+
 #### 3.12 Listing all archived employees : `c-archive-list`
 
-Shows a list of all archived employees' contact details in tCheck.
+Shows a list of all archived employees in _Employee Directory_ pane.
 
 Format: `c-archive-list`
 
-#### 3.13 Unarchiving an employee : `c-unarchive`
+<div markdown="span" class="alert alert-primary">
+
+**:information_source: Notes about displaying archived employees in tCheck:**<br>
+
+* Archived employees will be shown with a red color label, named as `[Archived]`, in front of their names. You may
+ refer to the screenshot below.<br>
 
-Unarchives the specified employee's contact detail from the tCheck's archived employee list.
+</div>
+
+Example:
+* Let's say you want to see an archived employee's information, you can follow these steps.
+
+Steps to see archived employees:
+1. Type `c-archive-list` into the _Command Box_.
+2. Press "Enter" to execute.
+
+
+Outcome:
+1. The _Result Display_ will show a success message.
+2. All archived employees will be listed in _Employee Directory_ pane,.
```

Comment:
Same issue here.

Reply:
It has been changed in the latest commit. Thanks!